app/src/main/kotlin/com/metrolist/music/viewmodels/ArtistViewModel.kt

```
@@ -169,7 +170,40 @@ class ArtistViewModel @Inject constructor(
+                                            .associate { it.id to it.duration }
+                                        if (durationById.isNotEmpty()) {
+                                            listOf(resolvedSections.first().copy(
+                                                items = resolvedSections.first().items.map { item ->
+                                                    if (item is SongItem && item.duration == null) {
+                                                        item.copy(duration = durationById[item.id] ?: item.duration)
+                                                    } else item
+                                                }
+                                            )) + resolvedSections.drop(1)
+                                        } else resolvedSections
+                                    } else resolvedSections
+                                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)